### ISSUE TITLE
[AXE-CORE]: LI MUST be contained by an UL

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#7727]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The GIBCT provider detail view has several accordion menus that are coded as `<div> + <li>` inside an `<ul>`. This is causing an axe error, and should be refactored as soon as possible. This current implementation breaks the relationship between the UL and its list items for screen readers. Screenshot attached below.

## Related Issues
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/7373

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Axe error no longer appears on future runs
- [ ] The expand/collapse accordions maintain current functionality (no regression)

## WCAG or Vendor Guidance (optional)

* [Deque Univeristy](https://dequeuniversity.com/rules/axe/3.5/listitem)
* [Info and Relationships:
Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2020-04-07 at 4 06 58 PM](https://user-images.githubusercontent.com/934879/78719901-4dbaf580-78ea-11ea-9be6-34141ccdabed.png)

**Comment by ksantiagomorgan6** [comment]
Pull Request: https://github.com/department-of-veterans-affairs/vets-website/pull/12296

**Comment by delli-gatti-michael** [comment]
Local testing complete

**Comment by delli-gatti-michael** [comment]
@1Copenut These changes are live in Staging and ready for your review

**Comment by 1Copenut** [comment]
@delli-gatti-michael I've taken a look and we're good to close. I wrote a follow on https://github.com/department-of-veterans-affairs/va.gov-team/issues/8392 for removing extra ARIA markup, because it's not needed for screen reader usability in this instance.
